### ISSUE TITLE
RE:Start Random Crashing

### ISSUE THREAD

**Issue by Raguto** [OpenKore/openkore#1214]
While botting on restart, I have been experiencing random crashes that don't seem to be triggered by anything that I can tell, it happens more often on a specific character of mine than the novice I have botting the training grounds. This begun happening after the latest maintenance and the couple of fixes to that said problem. I'm not sure what else I can provide to display the issue, I just get that openkore has crashed error that pops up. I will help in what ever way I can, anyone else having this issue?

![wxstart](https://user-images.githubusercontent.com/32392732/31020679-0e63876e-a50a-11e7-952f-da4e25444f8e.PNG)

**Comment by 10201244054** [comment]
Have u tried run from start.exe? Mine are running normally

**Comment by Raguto** [comment]
After botting overnight I find that the crashing only occurs when I'm here using my computer as opposed to leaving it alone. Still random.

**Comment by fadreus** [comment]
Try running at different computers does that help?
If kore crash usually you can check at errors.txt
But when software stopped working usually computer-related issue.
Probably RAM or CPU (since you mention it often crash when you were using computer rather than leaving it alone overnight)

Maybe if you can provide what's in errors.txt or the last thing in console (set config.txt to log console) or what recent changes have you made with your openkore and computers might help.

**Comment by Raguto** [comment]
@fadreus I don't have any other computer to test this on, I'm not running a huge bot operation. I also do not have any Errors.txt file in my folder, it's not been created? idk. 

I see that this seems to be a windows related issue as the crash is a windows error but this had only started after the previous maintenance on RO had occurred and the fixes were made to OpenKore.

Nothing else on the computer is crashing except for this, the only things that may have changed is when I restarted my computer and windows was doing a sneaky update that it shouldn't have been doing, do not know what it was either, I have windows updates turned off. I am using Windows 10 64-bit.

About the sleepTime thing, the OpenKore wxstart doesn't appear to be using very much of my CPU(in task manager) per each one, but when I'm botting I generally play on another character which as I can see the actual RO client uses a whole 30% of my CPU by itself.

I would also like to mention that I am using a program to split my single computer into two usable computers, with a program called Aster, which I suppose could be using a lot more of the CPU than I know about.

**Comment by fadreus** [comment]
There are few things you can try while waiting dev team to reply

- Set log console in config to see what's the last thing before crash (Don't set to hide any domain's messages).

- Each time it crash, let windows find the solution (to see if some error code printed).

If you don't have other PC to try, setup VM and run Kore there. If Kore didn't crash, it's your OS issue with recent update / mod. Can try run program compatibility and as administrator (Since your Windows did update something) unless some other people having this issues without using Win 10, you can ignore PC-related issue.

"In every murder case, even if victims are killed at random there must be some connection."
😆😆😆😆

**Comment by lututui** [comment]
Most likely, this is caused by CPU hitting 100% at some point

**Comment by Raguto** [comment]
A few updates on this issue. I have attempted to reproduce the issue by forcing my CPU to 100% and it wasn't causing the crash. I checked the console log for the corresponding characters that is having the issue and it happened when they were using the kafra. Though the crash has happened at other times like when dying or using fly wings or going into portals. It is random and so I do not think it would be OpenKore related. 

![100](https://user-images.githubusercontent.com/32392732/31123635-1c27b888-a817-11e7-8672-0a2dfbb4e444.PNG)

I guess I will have to do a couple other tests before figuring out what is going on.

**Comment by fadreus** [comment]
Or it just probably GM going around testing anti-bot, doing stuff to see if it work. Rather than crash, some people happen to just auto-exit and assume nothing is wrong, thus no similar issues reported (presumption 😆)
Hahahaha😂😂😂.. 

Btw, try run Kore on VM and then give the feedback.

**Comment by Raguto** [comment]
I am setting up a VM with Windows 8.1 as I write this, I'll get everything set up and see how it turns out, I can try it on Windows 10 as well after trying it on Windows 8.1 if that would be of further help.

**Comment by Raguto** [comment]
Oh man... I set up the VM with Windows 8.1 and ran my bot for a little while, anticipating a crash, hoping it would not come but it did, OpenKore crashed on the VM. Here's the console messages that were the most recent and a photo.

[Oct 3 12:49:35 2017.76] [ 71/ 14] Monster High Orc (0) attacks you (Dmg: Miss!) (Delay: 500ms)
[Oct 3 12:49:36 2017.12] [ 71/ 14] You attack Monster High Orc (0) (Dmg: 221*2 + 100) (Delay: 230ms)
[Oct 3 12:49:36 2017.48] You have gained 2713/1230 (0.06%/0.04%) Exp
[Oct 3 12:49:37 2017.69] Target died
[Oct 3 12:49:40 2017.68] Exiting...
[Oct 3 12:49:40 2017.70] macro plugin unloading, [Oct 3 12:49:40 2017.70] cleaning up
[Oct 3 12:49:40 2017.71] map unloaded.
[Oct 3 12:49:40 2017.72] [raiseSkill] Plugin unloading or reloading
[Oct 3 12:49:40 2017.72] [raiseStat] Plugin unloading or reloading
[Oct 3 12:49:40 2017.73] *** OpenKore what-will-become-2.1 ( version ctime:2017_09_28 ) - Custom Ragnarok Online client ***
***   http://www.openkore.com/   ***

![crash](https://user-images.githubusercontent.com/32392732/31138918-5ce01bfc-a846-11e7-9f18-e738359aa470.PNG)

**Comment by fadreus** [comment]
Oh.. Hmm, hard to explain but your best solution now is to use `start.exe`.

**Comment by Glikadin** [comment]
Dont use Windows 10 . Install Windows 7, God like this system.

**Comment by fadreus** [comment]
Yeah, I also don't recommend my customer using Win10. Most come ask for downgrade 😃

**Comment by Raguto** [comment]
It's not a Windows 10 issue though and my tests proved that, I've used Windows 7 and 8.1 plenty in the past and I think they are great.  @fadreus About the crashing though, you said that other people were having this same issue and no one has found why the problem is happening? Do you think it was something related to what Warpportal is doing on their end? Or is it my hardware. I can't really reproduce it since it is completely random and I think that is a huge part of the problem, I think I will use wxstart.exe when I'm at the computer since I can just restart the client. And just use start.exe when I go to sleep.

**Comment by fadreus** [comment]
Yes. From the issue they've posted some of the dev also found it weird & they can't reproduce it either.
All of the issues share 3 similarities; **Re:Start, wxstart.exe, random.**
Triggered by; **Running more than one wxstart.exe, different XKore, using item & skills.**
OS: Win 7

What I've found just a few things which it's didn't jeopardize any bot behavior but might be related.
If the expert didn't know the cause, I doubt I can be much of help any further 😌.

.
..
...

Just on the side note: There is also an issue in iRO with temporary solution during using portal `(route).`
Unknown Packets 0AB8 (which no one know what it does; ) when using skills & items (by my own experience).

These probably related to one another and these all **probably** part of iRO anti-bot project.
Rather then dumping the whole thing, iRO **probably** testing it slowly.
"Killing their victims slowly & painfully for their own joy" 😂😂😂
*_Damn, I watched too much investigation K-Drama LOL🤣🤣🤣_

Well, it just my analysis. I'm no expert but hope some idea crossed someone else mind fixing related stuff.
Hope this help something, someone, somewhere.😁
